tests/e2e/testcases/network/start/suite.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package network
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Local Network] Start", ginkgo.Ordered, func() {
+	ginkgo.AfterEach(func() {
+		err := commands.StopNetwork()
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, _ = commands.CleanNetwork()
```

Comment:
I believe clean stops network, so we don't need to stop it before cleaning

Reply:
addressed